tests/conftest.py

```diff
@@ -394,6 +394,16 @@ def do_patch(urls_to_responses=None):
         defaults = {
             "arborist/health": {"GET": ("", 200)},
             "arborist/auth/mapping": {"POST": ({}, "200")},
+            "arborist/policy": {"GET": ({
```

Comment:
I dont think you should add this in the defaults but instead pass as a parameter when calling mock_arborist_requests in the test function